### PR TITLE
Add inception docs

### PR DESCRIPTION
**Pull request by oke-aditya**
Add docs for inceptionv3. I think this is last one left.

Helps #5833

### CODE REVIEW

File: docs/source/models.rst [hunk 1]
```diff
@@ -457,7 +457,7 @@ ConvNeXt
     convnext_large
 
 SwinTransformer
---------
+---------------
```
Comment (oke-aditya) [inline docs/source/models.rst:460]: small bug

**Comment by oke-aditya** [comment]
#5833

File: torchvision/models/googlenet.py [hunk 2]
```diff
@@ -307,7 +307,7 @@ def googlenet(*, weights: Optional[GoogLeNet_Weights] = None, progress: bool = T
             weights are used.
         progress (bool, optional): If True, displays a progress bar of the
             download to stderr. Default is True.
-        **kwargs: parameters passed to the ``torchvision.models.squeezenet.GoogLeNet``
+        **kwargs: parameters passed to the ``torchvision.models.GoogLeNet``
```
Comment (oke-aditya) [inline torchvision/models/googlenet.py:310]: small bug spotted

File: torchvision/models/inception.py [hunk 3]
```diff
@@ -427,7 +427,8 @@ class Inception_V3_Weights(WeightsEnum):
 
 @handle_legacy_interface(weights=("pretrained", Inception_V3_Weights.IMAGENET1K_V1))
 def inception_v3(*, weights: Optional[Inception_V3_Weights] = None, progress: bool = True, **kwargs: Any) -> Inception3:
-    r"""Inception v3 model architecture from
+    """
+    Inception v3 model architecture from
     `"Rethinking the Inception Architecture for Computer Vision" <http://arxiv.org/abs/1512.00567>`_.
```
Comment (NicolasHug) [inline torchvision/models/inception.py:432]:
```suggestion
    `Rethinking the Inception Architecture for Computer Vision <http://arxiv.org/abs/1512.00567>`_.
```

**Review by NicolasHug — APPROVED** [review]
Thank you @oke-aditya !

**Pull request merged**